cylc/flow/scripts/lint.py

```diff
@@ -125,11 +125,35 @@
         'url': STYLE_GUIDE + 'trailing-whitespace',
         'index': 6
     },
-    re.compile(r'inherit\s*=\s*[a-z].*$'): {
+    # Look for families both from inherit=FAMILY and FAMILY:trigger-all/any
+    re.compile(r'(inherit\s*=\s*.*[a-z].*)|(\w[a-z]\w:.+?-a(ll|ny))'): {
         'short': 'Family name contains lowercase characters.',
         'url': STYLE_GUIDE + 'task-naming-conventions',
         'index': 7
     },
+    re.compile(r'({%.*%})|({{.*}})'): {
```

Comment:
Jinja2 templates can be split across lines like this
```
script = """
    {{
        x
    }}
"""
```
so maybe just match the beginning tags?
```suggestion
    re.compile(r'{[{%]'): {
```